26-register_publication/1625-creator_selects_reference_type_contribution_to_journal_and_selects_subtype_corrigendum.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Creator selects Reference type Contribution to journal and selects subtype Corrigendum
+
+    Scenario: Creator sees fields related to Reference subtype "Corrigendum"
+        Given Creator begins registering a Publication in the Wizard
+        When they navigate to the Reference tab
+        And they select the Reference type "Contribution to journal"
+        And they select the Reference subtype "Corrigendum"
+        Then they see fields:
+            | Search box for "Journal article" |
+            | DOI                              |
+            | Volume                           |
+            | Issue                            |
+            | Pages from                       |
+            | Pages to                         |
+            | Article number                   |
+        And they see a disabled field for Journal based on selected Journal article
```

Comment:
er denne egentlig nødvendig? Kan vi ikke "innlemme" denne i `Search box for Journal article`?

Reply:
tja, de er jo veldig knyttet sammen ja, men hvis vi fjerner det her baseres vi oss bare på wireframes/figma for at det skal vises da

Reply:
ja, sant